test/features/install_dataindex.feature

```diff
@@ -62,4 +62,31 @@ Feature: Kogito Data Index
     }
     """
 
-# External Kafka testing is covered in deploy_quarkus_service and deploy_springboot_service as it checks integration between Data index and KogitoRuntime
+#####
+
+  @postgresql
+  Scenario: Install Kogito Data Index with PostgreSQL
+    Given PostgreSQL instance "postgresql" is deployed within 3 minutes with configuration:
+      | username | myuser |
+      | password | mypass |
+      | database | mydb   |
+
+    When Install Kogito Data Index with 1 replicas with configuration:
+      | config | database-type | PostgreSQL               |
+      | config | infra         | kafka                    |
+      | runtime-env | QUARKUS_DATASOURCE_JDBC_URL     | jdbc:postgresql://postgresql:5432/mydb |
```

Comment:
using both URLs here but only one in scenario https://github.com/kiegroup/kogito-operator/pull/950/files#diff-4e0f459bc0fdab9e8a6bf7bce0c0c4b320fc5dfd7f751b4cc4e436c4286a7df0R433 ?